javascript/src/URDFLoader.js

```diff
@@ -73,7 +73,11 @@ class URDFLoader {
         const workingPath = THREE.LoaderUtils.extractUrlBase(urdf);
         const urdfPath = this.manager.resolveURL(urdf);
 
-        options = Object.assign({ workingPath }, options);
+        options = Object.assign({
+            workingPath,
+            parseVisual: true,
```

Comment:
These default values shouldn't be needed because they get specified in the `parse` function.